test/framework/src/main/java/org/elasticsearch/transport/nio/channel/ChannelFactory.java

```diff
@@ -130,10 +136,23 @@ ServerSocketChannel openNioServerSocketChannel(InetSocketAddress address) throws
             serverSocketChannel.configureBlocking(false);
             ServerSocket socket = serverSocketChannel.socket();
             socket.setReuseAddress(tcpReusedAddress);
-            serverSocketChannel.bind(address);
+            try {
+                serverSocketChannel.bind(address);
```

Comment:
same commetn as above move the `socket.setReuseAddress(tcpReusedAddress);` into the try block?